lib/seed/blood_pressure_seeder.rb

```diff
@@ -82,12 +84,14 @@ def call
         bp_id, recorded_at, patient_id = *row
         encounters << {
           blood_pressure_id: bp_id,
+          created_at: recorded_at,
           id: SecureRandom.uuid,
           device_created_at: recorded_at,
           device_updated_at: recorded_at,
           encountered_on: recorded_at,
           facility_id: facility.id,
           patient_id: patient_id,
+          updated_at: recorded_at,
```

Comment:
`Faker::Time.between(from: recorded_at, to: 1.day.ago)` has nanosecond randomization right? If that's the case this should be mostly solid.